.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: "Test"
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  packaging:
+    strategy:
+      fail-fast: true
+      matrix:
+        java: ["8", "11", "21"]
+    uses: "./.github/workflows/build.yml"
+    with:
+      version: "1.0.0"
+      java: "${{ matrix.java }}"
+  tests:
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: true
+      matrix:
+        java: ["8", "11", "21"]
```

Comment:
same here, we should waste too much of our github credits on test older versions.
we don't have any usage of this tool beside within SCT, and there we should focus on one single release 

if we'll have later ask for specific older version we would able to expend, or test it one off for some cases